felt-tool-api/src/main/resources/toolapi.mixins.json

```diff
@@ -1,15 +1,18 @@
 {
   "required": true,
   "minVersion": "0.8",
-  "package": "io.github.feltmc.feltapi.mixin.tool.common",
```

Comment:
is there anyway to define both mixin packages instead of just defining one and doing common,x and client.x? though that might be less clean overall hmm

Reply:
it is possible, just have to do 2 mixin.jsons